modules/services/dunst.nix

```diff
@@ -154,15 +168,15 @@ in {
         Service = {
           Type = "dbus";
           BusName = "org.freedesktop.Notifications";
-          ExecStart = "${cfg.package}/bin/dunst";
+          ExecStart = "${cfg.package}/bin/dunst ${cfg.commandOptions}";
           Environment = optionalString (cfg.waylandDisplay != "")
             "WAYLAND_DISPLAY=${cfg.waylandDisplay}";
         };
       };
     }
 
     (mkIf (cfg.settings != { }) {
-      xdg.configFile."dunst/dunstrc" = {
+      home.file."${cfg.configPath}" = {
```

Comment:
why change this ? I dont think the complexity is worth it.

Reply:
It also wouldn't work.

`home.file` expects a relative path and `xdg.configHome` is an absolute path because it uses `home.homeDirectory`.

Reply:
`home.file` accepts any path, absolute or relative. The only requirement is that the resolved absolute path is within $HOME.

Reply:
If preferred, even something like `xdg.configFile."dunst/${cfg.configFileName}"` would satisfy my use case.